libs/libnvme/Makefile

```diff
@@ -0,0 +1,55 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2023 Luca Barbato
+
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/kernel.mk
```

Comment:
seems unused?

Reply:
You are right, I assumed it was needed but it is not.